src/app/components/MediaLoader/configs/aresMedia.ts

```diff
@@ -103,6 +108,8 @@ export default ({
     playerConfig: {
       ...basePlayerConfig,
       autoplay: pageType !== 'mediaArticle',
+      ...(embedded && { insideIframe: true }),
+      ...(!embeddingAllowed && { externalEmbedUrl: '' }),
```

Comment:
In relation to the above comment, I think this could be removed if we take the approach of always setting `externalEmbedUrl` by default.

Reply:
Yh this helps keep it all cleaner.